boards/shields/tn0xxx/doc/index.rst

```diff
@@ -0,0 +1,76 @@
+.. _tn0xxx_shield:
+
+Kyocera Memory In Pixel TFT display generic shield
+##################################################
+
+Overview
+********
+
+This is a generic shield for Kyocera Memory In Pixel TFT LCD. It supports
+displays of TN0XXX type. These displays have an SPI interface and
+few other pins. Note that the SCS is active high for this display.
+
+Pins Assignment of the Generic Sharp memory Display Shield
+==========================================================
+
++---------------+---------------------------------------------------------+
+| Pin           | Function                                                |
++===============+=========================================================+
+| SCS           | Serial Slave Select                                     |
++---------------+---------------------------------------------------------+
+| SI            | Serial Data Input                                       |
++---------------+---------------------------------------------------------+
+| SCLK          | Serial Clock Input                                      |
++---------------+---------------------------------------------------------+
+| RST           | Reset                                                   |
++---------------+---------------------------------------------------------+
+| VCOM          | VCOM Inversion Polarity Input (VCOM can be controlled   |
+|               | through SW)                                             |
++---------------+---------------------------------------------------------+
+
+The zephyr display driver for TN0XXX only controls the SPI lines (SCS, SI, SCLK) - the power up and power down sequence is delegated to the user. This is because the datasheet specifies a power up and down sequence with timings specified between toggling of RST, VCOM and input SPI lines. This MIP display is typically used in low power applications, and as such having user control over the power down (or sleep) and power up (or wake up) options are important. Since the API to power on and off the device is not present in the zephyr driver display API, the control of these GPIOs per the datasheet is delegated to the user. Also note: VCOM = “L” is necessary when RST = ”H” otherwise the current consumption increases by shoot-through-current in panel (several mA).
```

Comment:
This line is to long, max 100 characters

Reply:
done! thank you for the feedback